### DISCUSSION TITLE
[HW2] Is it necessary to reuse our LSTM model from Assignment 1?

### DISCUSSION THREAD

**Discussion by iJinjin** [seominjoon/kaist-ai605#29] [Q&A] [answered]
Hello professor,
I was unfortunately unable to get my vanilla LSTM (or bi-LSTM) to work correctly when working on my first assignment. I am just a bit concerned that using my model from Assignment 1 will significantly affect the results for this second assignment. Should I still use my own LSTM model implementation? Thank you in advance!

**Comment by seominjoon** [comment, marked as answer]
You are allowed to use the `LSTM` module from PyTorch (but not  `MultiheadAttention` module).

**Reply by iJinjin** [reply]
Thank you professor!

**Reply by heyzude** [reply]
Professor, students who successfully implemented LSTM model in the first assignment are also allowed to use PyTorch's LSTM module?

**Reply by seominjoon** [reply]
Yes, you can use `LSTM` module whether you did it in Assignment 1 or not.

**Reply by heyzude** [reply]
Thank you :)
